Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -14,6 +14,13 @@ namespace Content.Server.Voting.Managers
 {
     public sealed partial class VoteManager
     {
+        public void SetupStandardVoteTypeEnableCVars()
```

Comment:
...which field initializer?

Reply:
Oh wait, you mean that of the dictionary. I didn't want to do that because it would put the standard vote type listing into VoteManager, which could lead to standard vote types not getting listed when they should be because someone doesn't know what needs to be updated.

Reply:
So I just realized those aren't mutually exclusive so long as I move the variable into this file